agenda/domain/src/main/java/com/gkp/agenda/domain/datasource/LocalAgendaDataSource.kt

```diff
@@ -0,0 +1,14 @@
+package com.gkp.agenda.domain.datasource
+
+import com.gkp.agenda.domain.model.AgendaItem
+import kotlinx.coroutines.flow.Flow
+
+interface LocalAgendaDataSource {
+    fun getAgendaItemsForDate(dateLong: Long): Flow<List<AgendaItem>>
+    suspend fun addOrSaveAgendaItem(agendaItem: AgendaItem)
+    suspend fun deleteAgendaItemById(id: String)
+    suspend fun deleteAllAgendaItems()
+    suspend fun updateAgendaItem(agendaItem: AgendaItem)
+    suspend fun getAgendaItemById(id: String): AgendaItem
```

Comment:
The same applies here. Does it not make more sense to return a nullable result (`AgendaItem?`) since the DB might not have an item that matches the `id` argument?